lib/tools.py

```diff
@@ -160,3 +152,235 @@ def reset_retriever(scope):
                 os.remove(os.path.join(HOME_DIR, 'connections.config'))
             except:
                 pass
+
+
+def is_valid_json(input_file):
+    """Validate a json file"""
+    try:
+        fp = open(input_file, 'r')
+        json.loads(fp.read())
+    except ValueError:
+        return False
+    return True
+
+
+def json2csv(input_file, output_file=None, header_values=None):
+    """Convert Json file to CSV
+
+    [
+    {"User": "Alex", "Country": "US", "Age": "25"}
+    ]
+    Alex,US,25
+
+    cross tab
+    [
+    {"User": "Alex", "Country": ["US","PT"], "Age": "25"},
+    ]
+
+    User,Country,Age
+    Alex,US,25
+    Alex,PT,25
+    """
+    try:
+        fp = open(input_file, 'r')
+    except IOError as e:
+        print "I/O error({0}): {1}".format(e.errno, e.strerror)
+
+    if not is_valid_json(input_file):
+        print "invalid json file"
```

Comment:
Let's use Python 3 print functions for all new print lines we're adding at this point.